compliance/collection/inventory/convert_test.go

```diff
@@ -0,0 +1,303 @@
+package inventory
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/uuid"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestInventoryConvert(t *testing.T) {
+	suite.Run(t, new(InventoryConvertTestSuite))
+}
+
+type InventoryConvertTestSuite struct {
```

Comment:
I think this can be unexported

Reply:
Not really sure why, but I've always made them unexported. Can't thing of why you'd need to use that struct outside of the package.